packages/dx-grid-core/src/plugins/paging-state/computeds.js

```diff
@@ -1,8 +1,16 @@
-export const paginate = (rows, pageSize, page) =>
-  rows.slice(pageSize * page, pageSize * (page + 1));
+const showAllPagesKey = 'all';
```

Comment:
Why have not you used upper case for this constant like we do in other cases (e.g. `DEFAULT_HEIGHT` or `MINIMAL_COLUMN_WIDTH`)?
I suggest renaming it in `ALL_PAGES_KEY` to make it look less alike a function.